src/cli-main.ts

```diff
@@ -8,11 +8,11 @@ const jwtDecode = require('jwt-decode');
 const CLI_CACHE_PATH = process.env.AKAMAI_CLI_CACHE_PATH;
 
 if (!CLI_CACHE_PATH) {
-  logAndExit("Error: AKAMAI_CLI_CACHE_PATH is not set.");
+  cliUtils.logAndExit(1, "ERROR: AKAMAI_CLI_CACHE_PATH is not set.");
```

Comment:
where are cliUtils defined?

I don't know how the logging is done, but to be compliant with CLIs we should report errors to stderr